src/htdocs/data/comcat/catalog/eqh/index.php

```diff
@@ -0,0 +1,13 @@
+<?php
+include '../_catalog.inc.php';
+?>
+
+<h3 id="additional">Additional Information</h3>
+
+<p>
+  <strong>Citation: Coffman, J.L., C.A. von Hake, and C.W. Stover, 1982, Earthquake History of the United States, U.S. Department of Commerce, National Oceanic and Atmospheric Administration and U.S. Department of Interior, Geological Survey, Boulder, Colorado, 1982, 258 pp.
+</p>
+
+<p>
+  This publication is a history of the prominent earthquakes in the United States from historical times through 1980." This is now generally considered to be superseded by the USHIS publication, except that this EQH publication includes events in Puerto Rico and the Virgin Islands that were not included in USHIS.
```

Comment:
There is an extra closing quote.

My first thought was this was a duplicate of USHIS, but I see it isn't.  Not sure if there should be links between the catalog pages?
https://earthquake.usgs.gov/data/comcat/catalog/ushis/